python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -192,7 +192,23 @@
 data_train_scaled.describe()
 
 # %% [markdown]
-# We can easily combine these sequential operations with a scikit-learn
+# We can use a jointplot to visualize the histograms and scatterplot of any
```

Comment:
I think before this you can add something to say that in the previous cell the mean of all the columns is close to 0 and the standard deviation of all the columns is close to 1 which is the `StandardScaler`'s purpose

Reply:
Done, thanks for the comments!